[profiler] add send/recv src/dst info

xw285cornell opened the pull request:
Summary: There is an ask to add src/dst to nccl trace. This feels like the easiest way to do - adding it to metadata seems to require plumbing a few stacks so will be more work

Test Plan: {F1128545195}

Reviewed By: davidberard98

Differential Revision: D50560692

aaronenyeshi reviewed (approved):
Looks reasonable to me. Accepting it to help unblock debugging.

xw285cornell committed:
[profiler] add send/recv src/dst info (#111811)

Summary:

There is an ask to add src/dst to nccl trace. This feels like the easiest way to do - adding it to metadata seems to require plumbing a few stacks so will be more work

Test Plan: {F1128545195}

Reviewed By: aaronenyeshi, fduwjj, davidberard98

Differential Revision: D50560692
